library/model/user.py

```diff
@@ -3,6 +3,10 @@
 from library.model.genre import Genre
 from library.persistence.storage import LibraryRepository
```

Comment:
The invoice import is placed in the middle of the class (Line 49 in the original code). We think it is better to put it at the beginning and remove the circular import.